config/env/env.go

```diff
@@ -57,7 +57,7 @@ func DecodeWithPrefix(conf interface{}, prefix string) {
 		envVal = strings.Split(envVal, ",")[0]
 
 		mapVal, exist := envMap[envVal]
-		if !exist || mapVal == "" {
```

Comment:
Why?
Define an empty env would even override Couper internal defaults on the config object.

Reply:
`DefaultSettings`

Reply:
It's right - ENV config has priority over other settings.